docs/server/installation.rst

```diff
@@ -7,106 +7,86 @@ These instructions will guide you through installing Pootle and its
 requirements in a virtual environment.
 
 If you only want to have a sneak peek of Pootle then the default configuration
-and the built-in server will suffice. But in case you want to deploy a real
-world server, :ref:`installing optional packages
-<optimization#optional_software>`, using a real database and :doc:`a proper web
-server <web>` is highly recommended.
+and the built-in server will suffice.
 
-The easiest way to test and install Pootle is by using pip, however,
-installating it straight from git sources is another viable approach.
+For a production deployment we recommend that you set up the following:
 
+- :ref:`install optional packages<optimization#optional_software>`
+- use either a :ref:`MySQL <mysql_installation>`
+  or :ref:`PostgreSQL <postgresql_installation>` database.
+- :doc:`make use of a front-end web server <web>`
 
-.. _installation#requirements:
 
-Requirements
-------------
+Before installing please ensure that you have all the
+:ref:`necessary requirements <requirements>`.
 
 
-.. _installation#hardware_requirements:
-
-Hardware Requirements
-^^^^^^^^^^^^^^^^^^^^^
-
-Your Pootle installation will need to be flexible enough to handle the
-translation load. The recommended hardware depends highly on the performance you
-expect, the number of users you want to support, and the number and size of the
-files you want to host.
-
-Whatever hardware you have, you will still benefit from performance improvements
-if you can :doc:`optimize your system <optimization>`.
+.. _installation#assumptions:
 
-Your disk space should always be enough to store your files and your Pootle
-database, with some extra space available.
+Setup assumptions
+-----------------
 
+We've made some assumptions in these instructions, adjust as needed:
 
-.. _installation#system_requirements:
+#. We're installing into :file:`~/dev/pootle`.
+#. We're using SQLite as its easy to setup.
+#. We're setting up the essential parts of Pootle including Redis, and RQ
+   Workers.
+#. This is a test installation on a single server, and not optimised for
+   production use.
+#. We're installing using :command:`pip`.
 
-System Requirements
-^^^^^^^^^^^^^^^^^^^
 
-To run Pootle you need a computer running:
+.. _installation#setup_system:
 
-- Linux
-- Mac OS X
+Installing system packages
```

Comment:
I suggest moving these instructions that are actually not about installing Pootle itself to *Pootle requirements* document (So outside of upgrading or installation instructions, but a link is placed in those).